resources/dicts/events/leader_den/success/other_clan.json

```
@@ -1405,11 +1746,33 @@
   },
   {
     "interaction_type": "antagonize",
-    "event_text": "At the Gathering, m_c forgoes a traditional speech in favor of retelling the tale of a legendary o_c_n leader... but with {PRONOUN/m_c/poss} own special graphically violent twist. o_c_n warriors cover their apprentices' ears, and even o_c_n's leader looks like they'll have nightmares.",
+    "event_text": "At the Gathering, o_c_n's leader repeatedly provokes and challenges c_n. m_c remains composed, not a whisker out of place, and asks o_c_n's leader to name a place and time for them to settle the score. {PRONOUN/m_c/poss/CAP} self-possession riles o_c_n's leader even more.",
+    "rel_change": -10,
+    "m_c": {
+      "trait": ["calm"],
+      "skill": ["FIGHTER,1"]
+    },
+    "player_clan_temper": ["any"],
+    "other_clan_temper": ["any"]
```

```suggestion
    "other_clan_temper": ["bloodthirsty", "proud"]
```
the rel change on this is so extreme i think it would be good to have a temper constraint as well

---

not sure which outcome this is? i may have edited it